src/components/InfoCard.tsx

```diff
@@ -61,29 +71,54 @@ const StyledInfoCard = styled.div`
   }
 
   @media (min-width: 768px) {
+    margin-bottom: 0rem;
+
     & > div > img {
       border-radius: 0rem;
     }
   }
+`;
 
-`
-
-const ButtonStyled = styled.button`
-  color: var(--oex-orange);
-  background-color: white;
+const ButtonContainer = styled.div`
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  justify-content: center;
+  width:100%;
   font: inherit;
-  padding: 1rem;
+  padding: 0rem;
   width: 100%;
-  border-radius: 0.2rem;
-  border: 0.09rem solid var(--oex-orange);
 
-  &:hover {
-    color: white ;
-    background-color: var(--oex-orange);
+  & > button {
```

Comment:
you have a "white" prop why should this have a background of white again?

Reply:
the fact that you're styling the buttons inside their container shows you are just using the CTA just because it's a CTA, you can modify the CTA to fit your need

Reply:
It has been moved into CTA. I created a custom style that inherits from CTA